swagger_to_sdk_config.json

```diff
@@ -26,10 +26,10 @@
       "build_dir": "azure-mgmt-authorization"
     },
     "batch.management": {
-      "swagger": "arm-batch/2017-01-01/swagger/BatchManagement.json",
+      "swagger": "arm-batch/2017-05-01/swagger/BatchManagement.json",
       "autorest_options": {
         "Namespace": "azure.mgmt.batch",
-        "PackageVersion": "3.0.1"
+        "PackageVersion": "4.0.0"
```

Comment:
could you add "build_dir": "azure-mgmt-batch" just after "output_dir" and restart the generate script?